src/2025h2/bridge-between-incrementals.md

```diff
@@ -0,0 +1,99 @@
+# Bridge between incremental caches for all tools
+
+| Metadata         |                                                                                  |
+|:-----------------|----------------------------------------------------------------------------------|
+| Point of contact | @blyxyas                                                                         |
+| Teams            | (probably WG-incremental but waiting approval by them)                           |
+| Task owners      | @blyxyas                                                                          |
+| Status           | Proposed                                                                         |
+| Tracking issue   |                                                                                  |
+| Zulip channel    |                                                                                  |
+
+## Summary
+
+The goal of this project goal is to make a compatibility layer to share incremental compilation information between
+`cargo check`, `cargo build`, and any other external tool that might benefit from sharing incremental information
+such as `cargo clippy`.
+
+## Motivation
+
+Redoing work between `cargo check`, `cargo build` and other commands such as `cargo clippy` is one of the greatest pain
+points in the realm of compiler performance. Being able to reuse this information would speed up by a lot both
```

Comment:
For me, this comes across as a pretty strong statement and yet it doesn't match my experience or those I've talked to.